hono-vite-jsx/README.md

```diff
@@ -2,32 +2,20 @@
 
 Author: [@MathurAditya724](https://github.com/MathurAditya724)
 
-This example shows how to use Hono's [Client Components](https://hono.dev/docs/guides/jsx-dom) with Vite and integrate it with the Hono API.
+This example shows how to use Hono's [jsx-dom](https://hono.dev/docs/guides/jsx-dom) with Vite and integrate it with the Hono API.
 
-## UI Commands
+## Commands
 
-Run the Vite development server
+Run the `vite` dev server
 
 ```bash
-pnpm run ui:dev
-```
-
-Build the UI
-
-```bash
-pnpm run ui:build
+pnpm run dev
```

Comment:
We need to unify the package managers used in this project, but it is not related to this PR. I think `npm` is a good choice because it's general.